docs/content/datasources.md

```diff
@@ -698,6 +698,8 @@ This table describes the currently-supported authentication mechanisms and how t
 | [`userpass`](https://www.vaultproject.io/docs/auth/userpass.html) | Environment variables `$VAULT_AUTH_USERNAME` and `$VAULT_AUTH_PASSWORD` must be set to the appropriate values.<br/> If the back-end is mounted to a different location, set `$VAULT_AUTH_USERPASS_MOUNT`. |
 | [`token`](https://www.vaultproject.io/docs/auth/token.html) | Determined from either the `$VAULT_TOKEN` environment variable, or read from the file `~/.vault-token` |
 | [`aws`](https://www.vaultproject.io/docs/auth/aws.html) | The env var  `$VAULT_AUTH_AWS_ROLE` defines the [role](https://www.vaultproject.io/api/auth/aws/index.html#role-4) to log in with - defaults to the AMI ID of the EC2 instance. Usually a [Client Nonce](https://www.vaultproject.io/docs/auth/aws.html#client-nonce) should be used as well. Set `$VAULT_AUTH_AWS_NONCE` to the nonce value. The nonce can be generated and stored by setting `$VAULT_AUTH_AWS_NONCE_OUTPUT` to a path on the local filesystem.<br/>If the back-end is mounted to a different location, set `$VAULT_AUTH_AWS_MOUNT`.|
+| [`jwt`](https://www.vaultproject.io/docs/auth/jwt) | `$VAULT_AUTH_JWT_TOKEN` and `$VAULT_AUTH_JWT_ROLE` must be set.<br/> If the back-end is mounted to a different location, set `$VAULT_AUTH_JWT_MOUNT`. |
+| [`kubernetes`](https://www.vaultproject.io/docs/auth/kubernetes) | `$VAULT_AUTH_KUBERNETES_ROLE` must be set to the role to log into vault with. Gomplate will authenticate using the Pod service account token by default. To use a different token, set `$VAULT_AUTH_KUBERNETES_TOKEN`.<br/> If the back-end is mounted to a different location, set `$VAULT_AUTH_KUBERNETES_MOUNT`. |
```

Comment:
These are great, but should be in order of precedence - currently your implementation has the EC2 (`aws`) role last. I think that's the right order, so these docs just need to change. Also I've fixed just a couple minor typos:

```suggestion
| [`jwt`](https://www.vaultproject.io/docs/auth/jwt) | `$VAULT_AUTH_JWT_TOKEN` and `$VAULT_AUTH_JWT_ROLE` must be set.<br/>If the back-end is mounted to a different location, set `$VAULT_AUTH_JWT_MOUNT`. |
| [`kubernetes`](https://www.vaultproject.io/docs/auth/kubernetes) | `$VAULT_AUTH_KUBERNETES_ROLE` must be set to the role to log into Vault with. Gomplate will authenticate using the Pod's service account token by default. To use a different token, set `$VAULT_AUTH_KUBERNETES_TOKEN`.<br/>If the back-end is mounted to a different location, set `$VAULT_AUTH_KUBERNETES_MOUNT`. |
| [`aws`](https://www.vaultproject.io/docs/auth/aws.html) | The env var  `$VAULT_AUTH_AWS_ROLE` defines the [role](https://www.vaultproject.io/api/auth/aws/index.html#role-4) to log in with - defaults to the AMI ID of the EC2 instance. Usually a [Client Nonce](https://www.vaultproject.io/docs/auth/aws.html#client-nonce) should be used as well. Set `$VAULT_AUTH_AWS_NONCE` to the nonce value. The nonce can be generated and stored by setting `$VAULT_AUTH_AWS_NONCE_OUTPUT` to a path on the local filesystem.<br/>If the back-end is mounted to a different location, set `$VAULT_AUTH_AWS_MOUNT`.|
```